helm/templates/daemonset.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ .Values.merbridge.fullname }}
+  namespace: {{ .Values.merbridge.namespace }}
```

Comment:
`{{ .Release.Namespace }}` may be better.

Reply:
question is that if we use {{ .Release.Namespace }}, when we generate templates, how to handle the generated namespace?

Reply:
Any flags or commands to specific the release name/namespace when executing helm template?